packages/react/src/themes/teams/components/TextArea/textAreaVariables.ts

```diff
@@ -1,11 +1,41 @@
-import { ResizeProperty } from 'csstype'
+import { pxToRem } from '../../../../lib'
 
 export interface TextAreaVariables {
+  backgroundColor: string
+  backgroundColorInverted: string
+  backgroundColorDisabled: string
```

Comment:
```suggestion
  disabledBackgroundColor: string
```

As I remember prop name should be first